wpilibj/src/main/java/edu/wpi/first/wpilibj/PIDController.java

```diff
@@ -685,8 +685,8 @@ public double getError() {
    * number of iterations to average with setToleranceBuffer() (defaults to 1). getAvgError() is
    * used for the onTarget() function.
    *
+   * @return the current average of the error
```

Comment:
Why doesn't it line these up?

Reply:
One is `@return` and the other is `@deprecated`. The bullet list in the docs for `IterativeRobotBase` have been re-indented.

Reply:
I guess it only lines up tags with the same "@" string.